pkg/codegen/internal/test/program_driver.go

```diff
@@ -49,8 +49,11 @@ var programTests = []programTest{
 	{
 		Name:        "aws-fargate",
 		Description: "AWS Fargate",
+
+		// TODO[pulumi/pulumi#8440]
```

Comment:
So this is back to where master was, but now with a note linking that we have an issue. Go code does not compile here on master.